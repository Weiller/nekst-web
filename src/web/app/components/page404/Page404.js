import React from 'react'

const Page404 = () => {
    return (<div> Página não encontradsa</div>)
}

export default Page404