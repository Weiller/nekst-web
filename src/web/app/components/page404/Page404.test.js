import React from 'react'
import {shallow} from 'enzyme'
import Page404 from "./Page404";
import toJson from 'enzyme-to-json';


describe('Teste do componente - Page 404', () => {

    it('test render', () => {
        const component = shallow(<Page404/>)
        console.log(toJson(component))
        expect(toJson(component)).toMatchSnapshot()
    });
})