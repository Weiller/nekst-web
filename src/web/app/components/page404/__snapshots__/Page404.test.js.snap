// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Teste do componente - Page 404 test render 1`] = `
<div>
   Página não encontrada
</div>
`;
